frontend/src/types.ts

```diff
@@ -1059,15 +1059,20 @@ export type SessionRecordingSnapshotParams =
           source: 'blob_v2'
           blob_key?: string
       }
+    | {
+          source: 'blob_v2'
+          start_blob_key?: string
+          end_blob_key?: string
+      }
     | {
           source: 'realtime'
-          // originally realtime snapshots were returned in a different format than blob snapshots
-          // since version 2024-04-30 they are returned in the same format
-          version: '2024-04-30'
```

Comment:
this code was only to avoid breaking existing sessions
and if someone has a running posthog session since April 2024... they can refresh